cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_search_overviews.spec.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+const workspaceName = `test_workspace_search_${Math.random()
+  .toString(36)
+  .substring(7)}`;
+let workspaceDescription = 'This is a  search workspace description.';
+let workspaceId;
+let datasourceId;
+let workspaceFeatures = ['use-case-search'];
+
+const MDSEnabled = Cypress.env('DATASOURCE_MANAGEMENT_ENABLED');
+
+if (Cypress.env('WORKSPACE_ENABLED')) {
+  const createWorkspace = (datasourceId) => {
+    cy.createWorkspace({
+      name: workspaceName,
+      description: workspaceDescription,
+      features: workspaceFeatures,
+      settings: {
+        permissions: {
+          library_write: { users: ['%me%'] },
+          write: { users: ['%me%'] },
+        },
+        ...(datasourceId ? { dataSources: [datasourceId] } : {}),
+      },
+    }).then((value) => {
+      workspaceId = value;
+    });
+  };
```

Comment:
```suggestion
  const createWorkspace = (datasourceId) => {
    cy.createWorkspace({
      name: workspaceName,
      description: workspaceDescription,
      features: workspaceFeatures,
      settings: {
        permissions: {
          library_write: { users: ['%me%'] },
          write: { users: ['%me%'] },
        },
        ...(datasourceId ? { dataSources: [datasourceId] } : {}),
      },
    }).then((value) => {
      workspaceId = value;
    });
  };
```
We can move the function to a util function maybe?

Reply:
`cy.createWorkspace` already been a util function, and this `createWorkspace` function just utilized the `cy.createWorkspace` to create workspace with and without datasource.  To move it to a common place as util function may not benefits other test cases.